src/pages/docs/creators/ray/troubleshooting.md

```diff
@@ -1,13 +1,72 @@
 ---
 title: Ray on Golem troubleshooting
-description: Comprehensive guide for resolving common Ray on Golem issues, including log file analysis and problem solving techniques.
+description: Comprehensive guide for resolving common Ray on Golem issues, including log file analysis and problem-solving techniques.
 pageTitle: Ray on Golem Troubleshooting Guide - Identify and Resolve Common Issues
 
 type: troubleshooting
 ---
 
 # Ray on Golem troubleshooting
 
+{% troubleshooting %}
+
+## What is going on with my cluster - status and dashboard 
+ 
+
+{% problem /%}
+
+Often you will find yourself wondering "How is my cluster doing?"
+
+{% solution %}
+
+Ray offers two tools to inspect the cluster state: the status and the dashboard.
+
+Ray status shows information about nodes constituting the cluster, the total resources of the cluster, and their current usage.
+
+The `ray status` command needs to be executed on the head node:
+```bash
+ray exec golem-cluster.yaml 'ray status'
+```
+```
+Ray On Golem webserver
+  Not starting webserver, as it's already running
+Fetched IP: 192.168.0.3
+======== Autoscaler status: 2023-11-28 08:09:48.391906 ========
+Node status
+---------------------------------------------------------------
+Healthy:
+ 1 ray.head.default
+ 1 ray.worker.default
+Pending:
+ ray.worker.default, 5 launching
+Recent failures:
+ (no failures)
+
+Resources
+---------------------------------------------------------------
+Usage:
+ 2.0/2.0 CPU
+ 0B/5.34GiB memory
+ 0B/2.61GiB object_store_memory
+
+Demands:
+ {'CPU': 1.0}: 385+ pending tasks/actors
+Shared connection to 192.168.0.3 closed.
+
+```
+
+The Ray dashboard visualizes the nodes and what happens to them. 
+
+Run `ray dashboard golem-cluster.yaml` to forward your cluster's dashboard to [http://localhost:8265](http://localhost:8265).
+Keep it on to keep the tunnel active.
```

Comment:
```suggestion
Keep it running in the background so that the tunnel stays active as long as you need it.
```

(I mostly wanted to prevent the repetition of `keep`)